lib/licensee/license_meta.rb

```diff
@@ -53,5 +58,9 @@ def [](key)
       key = 'spdx_id' if key == 'spdx-id'
       super(key)
     end
+
+    def spdx_license_url(spdx_id)
+      return if spdx_id.nil?
```

Comment:
Should this return something if it's not null?

Reply:
whoops, that's not used, must be leftover from experiment with different approach. will rm.